server/routes/api/push.js

```diff
@@ -13,8 +13,8 @@ function sortByDate(a, b) {
 // Handles get requests for /v1/push
 router.get('/', function(req, res) {
     authMiddleware('admin', 'api', true, function() {
-        PushNotification.find({}, '-_id -__v')
-            .byIsPublic()
+        PushNotification.find()
+            .byIsPublic(req.query.since)
```

Comment:
does this default to 0 if its not provided

Reply:
If you jump into the `byIsPublic` query helper, you'll see that if undefined or false is provided to the method, it'll default to 0.